### DISCUSSION TITLE
[For Research Purpose] Sample Implementation with Two different dataset as Tasks.

### DISCUSSION THREAD

**Discussion by MagicHealer** [ContinualAI/avalanche#1154] [General]
I am currently working on Research right now about CL and I will be using Avalanche to make my work easier. In connection to this, does anyone here have a sample implementation of CL using Avalanche Library using two different Datasets as Tasks? Example, Imagenet for First task and CUB dataset for second task? Thank you.

**Comment by AntonioCarta** [comment]
you can use [dataset_benchmark](https://avalanche-api.continualai.org/en/v0.2.1/generated/avalanche.benchmarks.generators.dataset_benchmark.html#avalanche.benchmarks.generators.dataset_benchmark).
